astroquery/utils/tests/test_system_tools.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Test `astroquery.utils.system_tools`.
+
+"""
+
+# STDLIB
+import gzip
```

Comment:
A bit bike shedding, but now that gzip is assumed to be optional, we need to make sure to skip the test if it's not available.